content/guides/python/tracers.md

```diff
@@ -0,0 +1,97 @@
+- Tracing credentials.
+- Sampling strategy.
+
+For example, initializing the `Tracer` implementation of Jaeger might look like this:
+
+```python
+import logging
+from jaeger_client import Config
+
+def init_tracer(service):
+    logging.getLogger('').handlers = []
+    logging.basicConfig(format='%(message)s', level=logging.DEBUG)
+
+    config = Config(
+        config={
+            'sampler': {
+                'type': 'const',
+                'param': 1,
+            },
+            'logging': True,
+        },
+        service_name=service,
+    )
+
+    # this call also sets opentracing.tracer
+    return config.initialize_tracer()
+```
+
+To use this instance, replace `opentracing.tracer` with `init_tracer(...)`, like so:
+
+`tracer = init_tracer('hello-world')`
+
+Complete tutorial is available [here](https://github.com/yurishkuro/opentracing-tutorial/tree/master/python/lesson01).
+
+### Starting a new Trace
+The `Tracer` instance obtained can be used to manually create `Span` that will start a new trace. This can be done in two ways:
+
+1. `Tracer.start_active_span()` creates a new `Span` and automatically activates it.
+
+2. `Tracer.start_span()` creates a new `Span`
+
+`Tracer.start_span()` and `Tracer.start_active_span()` will automatically use the current active `Span` as a parent, unless the programmer passes a specified parent context or sets `ignore_active_span=True`.
+
+A new `trace` can be started like so:
+
+```Python
+
+# Manual activation of the Span.
+span = tracer.start_span(operation_name='someWork')
+with tracer.scope_manager.activate(span, True) as scope:
+    # Do things.
+
+# Automatic activation of the Span.
+# finish_on_close is a required parameter.
+with tracer.start_active_span('someWork', finish_on_close=True) as scope:
+    # Do things.
+
+```
+
+### Accessing the Active Span
+For getting/setting the current active `Span` in the used request-local storage, OpenTracing requires that every `Tracer` contains a `ScopeManager` that grants access to the active `Span` through a `Scope`. Any `Span` may be transferred to another task or thread, but not `Scope`. This is implemented as follows:
+
+```Python
+# Access to the active span is straightforward.
+scope = tracer.scope_manager.active()
+if scope is not None:
+    scope.span.set_tag('...', '...')
+```
+
+### Propagating a Trace with Inject/Extract
+In order to trace across process boundaries in distributed systems, services need to be able to continue the trace injected by the client that sent each request. OpenTracing allows this to happen by providing inject and extract methods that encode a span's context into a carrier.
+The `inject` method allows for the `SpanContext` to be passed on to a carrier. The `extract` method does the exact opposite. It extracts the `SpanContext` from the carrier.
+
+Implemented examples of inject/extract and details can be found [here](./inject-extract).
+
+![Trace Propagation](/img/overview:tracers/Extract.png)
```

Comment:
can you add a basic example using http here? The Inject/Extract docs should go into all the options, but a basic example would be good to have.